books/free-programming-books-langs.md

```diff
@@ -1967,6 +1967,7 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Python Programming](https://upload.wikimedia.org/wikipedia/commons/9/91/Python_Programming.pdf) - Wikibooks (PDF) (2.6)
 * [Python Programming And Numerical Methods: A Guide For Engineers And Scientists](https://pythonnumericalmethods.berkeley.edu/notebooks/Index.html) - Qingkai Kong, Timmy Siauw, Alexandre Bayen (3.7)
 * [Python Tutorial](https://www.tutorialspoint.com/python/) - Tutorials Point (HTML, PDF)
+* [Scientific Visualization: Python + Matplotlib](https://github.com/rougier/scientific-visualization-book/blob/master/pdf/book.pdf) - Nicolas P. Rougier (PDF)
```

Comment:
```suggestion
* [Scientific Visualization: Python + Matplotlib](https://github.com/rougier/scientific-visualization-book) - Nicolas P. Rougier (PDF, Markdown)
```
Please provide the link to the repo. So the user can download the book from there

Reply:
Ok, I will make the URL link change.  Originally I had the direct link to the pdf, but a shorter link to the repo is good.

I am not sure about adding "Markdown" to the formats list.  The book is not written in Markdown format.